cli_tools/common/disk/inspect.go

```diff
@@ -63,50 +71,197 @@ func NewInspector(wfAttributes daisycommon.WorkflowAttributes, network string, s
 	daisycommon.SetWorkflowAttributes(wf, wfAttributes)
 	wf.Vars["network"] = daisy.Var{Value: network}
 	wf.Vars["subnet"] = daisy.Var{Value: subnet}
-	return &defaultInspector{wf}, nil
+	return &bootInspector{[]string{}, &defaultDaisyWorker{wf}}, nil
 }
 
-// defaultInspector implements disk.Inspector using a Daisy workflow.
-type defaultInspector struct {
-	wf *daisy.Workflow
+// bootInspect implements disk.Inspector using the Python boot-inspect package,
```

Comment:
bootInsector to match the name of the struct?

Also, would bootDiskInspector be more appropriate?

Reply:
Fixed the comment!

Regarding `bootDiskInspector`, I don't have a huge preference. Since the package name is `disk` already, I was thinking it'd be implicit. More in the next comment on this....